frontend/plan/Dockerfile

```diff
@@ -9,6 +9,7 @@ COPY package*.json /app/
 RUN npm install
 
 COPY . /app/
+COPY ../shared-components/src /app/src/components
```

Comment:
Just tested a docker build locally and this doesn't work... will need to find a workaround. Probably eventually we should make this an NPM package that we publish as part of the build step?

Reply:
It doesn't need to be a published npm package, we can just put a github link in our package.json. Also we could also use git submodules as another alternative.